frontend/packages/console-shared/src/components/catalog/catalog-view/CatalogToolbar.tsx

```diff
@@ -61,7 +61,7 @@ const CatalogToolbar = React.forwardRef<HTMLInputElement, CatalogToolbarProps>(
       <div className="co-catalog-page__header">
         <div className="co-catalog-page__heading text-capitalize">{title}</div>
         <div className="co-catalog-page__filter">
-          <div ref={inputRef}>
+          <div ref={inputRef} className="co-catalog-page__searchfilter-div">
```

Comment:
Just a nit, but there is no reason to define a HTML element type here. If we change this to a `<p>` this style should still work. So let us drop the `-div` here, or?

```suggestion
          <div ref={inputRef} className="co-catalog-page__searchfilter">
```

Reply:
Added the changes